### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by joeynyan**
Fixed a few grammar mistakes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ ff 34 10 0a
 ff 
 ```
 
-The first 8 byte is always ` 19 81 03 38 00 92 00 31 `, I'm not sure if this differs in different Joycons because I only took apart one.
+The first 8 byte is always ` 19 81 03 38 00 92 00 31 `, I'm not sure if this differs in different Joycons because I have only took apart one.
```
Comment (nloveladyallen) [inline README.md]: "took" or "have taken", not "have took"

File: README.md [hunk 2]
```diff
@@ -128,5 +128,5 @@ Right now I'm working on spoofing Joycon with a microcontroller, it's basically
 
 ## Ending remarks
 
-Right now I only took apart one left Joycon and yet to tear into the console itself, because I want to finish the Zelda first. But so far it looks like Nintendo make some really weird design decisions both in software and in hardware, probably to make what I'm doing more difficult. Anyway, I'll update this from time to time when I have new discoveries. Please share if you find this useful.
+Right now I have only took apart one left Joycon and have yet to tear into the console itself, because I want to finish the Zelda first. But so far it looks like Nintendo has made some really weird design decisions both in software and in hardware, will probably make what I'm doing more difficult. Anyway, I'll update this from time to time when I have new discoveries. Please share if you find this useful.
```
Comment (nloveladyallen) [inline README.md]: Comma splices are fine in informal language, but it's kinda the antithesis of a pedantic grammar PR

Comment (RafaelELopez) [inline README.md]: "which will probably" or "that will probably" are both better than "will probably", which on its own is incorrect. Better yet, leave "probably to" as it was.

**Comment by CTCaer** [comment]
@dekuNukem 
I've added the suggestions to my latest pull request.

Please close this abandoned pull request.

**Pull request closed without merging**